core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -22,6 +22,7 @@
 import io.trino.spi.block.Block;
 import io.trino.spi.block.BlockBuilder;
```

Comment:
there seems to be regression for `BenchmarkGroupByHashOnSimulatedData.groupBy    .9      BIGINT_1K_GROUPS  avgt   10   5,614 ± 0,019  ns/op` case

Reply:
For me this looks more like a flake. Should I run a full set of microbenchmarks?

Reply:
> Should I run a full set of microbenchmarks?

Just this one? I dont think that single case is very relevent though